wire-ios-sync-engine/Source/UserSession/ZMUserSession/AppVersionMigrations/AppVersionMigration_4_3_0.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireCoreCrypto
+import WireDomain
+
+/// **Issue:**: Use the new CC method to update the database key.
+struct AppVersionMigration_4_3_0: AppVersionMigration {
+
+    let version: SemanticVersion = "4.3.0"
+    let coreCryptoProvider: CoreCryptoProviderProtocol
+
+    func perform() async throws {
+        try await coreCryptoProvider.updateDatabaseKey()
+        coreCryptoProvider.reset()
```

Comment:
not a blocker, but you could perform the reset inside the `updateDatabasKey()` method.

Reply:
done